src/main/java/com/github/tomakehurst/wiremock/matching/EqualToUrlEncodedFormPattern.java

```diff
@@ -10,32 +16,39 @@
 import java.util.List;
 import java.util.Map;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.base.Objects;
-
-import org.apache.commons.lang3.StringUtils;
+import static com.google.common.collect.FluentIterable.from;
 
 public class EqualToUrlEncodedFormPattern extends StringValuePattern {
 
     private final Map<String, String> expected;
     private final String urlEncodedForm;
     private final String charset;
+    private final List<StringValuePattern> captureFormParameters;
 
-    public EqualToUrlEncodedFormPattern(@JsonProperty("equalToUrlEncodedForm") String urlEncodedForm) {
+    public EqualToUrlEncodedFormPattern(@JsonProperty("equalToUrlEncodedForm") String urlEncodedForm,
+                                        @JsonProperty("captureFormParameters") List<StringValuePattern> captureFormParameters) {
         super(urlEncodedForm);
+        if (captureFormParameters == null)
+            captureFormParameters = Collections.<StringValuePattern>singletonList(new AnythingPattern());
         this.charset = StandardCharsets.ISO_8859_1.name();
-        this.expected = parse(urlEncodedForm);
+        this.expected = parse(urlEncodedForm, captureFormParameters);
         this.urlEncodedForm = normalizedString(expected);
+        this.captureFormParameters = captureFormParameters;
     }
 
     @JsonProperty("equalToUrlEncodedForm")
     public Object getEqualToUrlEncodedForm() {
         return urlEncodedForm;
     }
 
+    @JsonProperty("captureFormParameters")
```

Comment:
When the recordings are saved their JSON will include the parameter filters they were recorded with. That way we can apply the same filters when matching requests on playback.